scripts/ci/install-build-system

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import os
+import glob
+from subprocess import check_call
+import shutil
+
+_dname = os.path.dirname
+
+REPO_ROOT = _dname(_dname(_dname(os.path.abspath(__file__))))
+os.chdir(REPO_ROOT)
+
+
+def run(command):
+    return check_call(command, shell=True)
+
+run('pip install --no-build-isolation build==0.7.0')
```

Comment:
It would be nice if we can store this dependency pin in a requirements file to make it easy to locate and update